my-playbook/roles/nginx/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+nginx_user: www-data
+nginx_group: www-data
+nginx_listen_port: 80
+nginx_server_name: "localhost"
+nginx_server_root: "/var/www/html"
+nginx_index_file: "index.html"
+nginx_template_file: "nginx.conf.j2"
```

Comment:
Can we have env level common var file? 
Like not to mantain diff var files for each component, WDYT?